### ISSUE TITLE
Hazelcast warnings

### ISSUE THREAD

**Issue by tt-le** [OpenLiberty/guide-sessions#96]
### OpenJDK 11
`mvn liberty:dev` from `finish` directory with OpenJDK11
`Launching defaultServer (Open Liberty 20.0.0.10/wlp-1.0.45.cl201020200915-1100) on OpenJDK 64-Bit Server VM, version 11.0.8+10 (en_CA)` produces the following warnings messages.

```bash 
[INFO] [WARNING ] Hazelcast is starting in a Java modular environment (Java 9 and newer) but without proper access to required Java packages. Use additional Java arguments to provide Hazelcast access to Java internal API. The internal API access is used to get the best performance results. Arguments to be used:
[INFO]  --add-modules java.se --add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.nio=ALL-UNNAMED --add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.management/sun.management=ALL-UNNAMED --add-opens jdk.management/com.sun.management.internal=ALL-UNNAMED

[INFO] WARNING: An illegal reflective access operation has occurred
[INFO] WARNING: Illegal reflective access by com.hazelcast.internal.networking.nio.SelectorOptimizer (file:/Users/tien-thanhle/.m2/repository/com/hazelcast/hazelcast/3.12.6/hazelcast-3.12.6.jar) to field sun.nio.ch.SelectorImpl.selectedKeys
[INFO] WARNING: Please consider reporting this to the maintainers of com.hazelcast.internal.networking.nio.SelectorOptimizer
[INFO] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[INFO] WARNING: All illegal access operations will be denied in a future release
```

### OpenJDK 8
`mvn liberty:dev` from `finish` directory with OpenJDK8
`Launching defaultServer (Open Liberty 20.0.0.10/wlp-1.0.45.cl201020200915-1100) on OpenJDK 64-Bit Server VM, version 1.8.0_265-b01 (en_CA)`

No warnings

**Investigating with HazelCast 4.0.3**

**Comment by gkwan-ibm** [comment]
@tt-le would you try Java8, warning will not be shown?
Also, would you try hazelcast 4.0.3?

**Comment by gkwan-ibm** [comment]
close
- schema warning was fixed by #98 
- the java warnings should be Hazelcast issue
